packages/api/src/services/market-prices.ts

```diff
@@ -72,7 +73,7 @@ export default function (config: Config, appState: Dependencies) {
   // we can try to price all known erc20 addresses. Some will fail. Also this endpoint does not return a timestamp
   // so we will just set one from our query time.
   async function update(timestampS = nowS()) {
-    const addresses = Array.from(collateralAddresses.values()).concat(Array.from(syntheticAddresses.values()));
+    const addresses = Array.from(new Set(Array.from(collateralAddresses).concat(Array.from(syntheticAddresses))));
```

Comment:
make sure we dont somehow have duplicate addresses

Reply:
nit: for readability, should we use lodash uniq? https://lodash.com/docs/4.17.15#uniq